code/game/objects/items/stacks/medical.dm

```diff
@@ -49,6 +49,15 @@ Contains:
 				to_chat(user, "<span class='warning'>You can't apply [src] through [H.wear_suit]!</span>")
 				return 1
 
+		if(affecting.status & ORGAN_LIFELIKE)
+			if(!(affecting.brute_dam || affecting.burn_dam))
+				to_chat(user, "<span class='notice'> [M] seems healthy, there are no wounds to treat! </span>")
+				return 1
+
+			to_chat(user, "<span class='notice'>You apply [src], but it doesn't seem to have any effect...</span>")
```

Comment:
```suggestion
			to_chat(user, "<span class='notice'>You apply \the [src], but it doesn't seem to have any effect...</span>")
```

Reply:
No, `\the` is added automatically unless you use `[src.name]` instead of `[src]`.

Reply:
Tried that, it results in 'the the ointment' and stuff like that.